src/quacc/settings.py

```diff
@@ -126,6 +126,9 @@ class QuaccSettings(BaseSettings):
     # ---------------------------
     # ESPRESSO Settings
     # ---------------------------
+    ESPRESSO_BIN_DIR: Path = Field(
+        Path(), description="Base path to the espresso binaries."
```

Comment:
Please use `Path.cwd()` to get the absolute path.

Reply:
For now, it wouldn't work to put Path.cwd() for the default case we want it to be 'pw.x' (assuming it is in the path) not /some/random/path/cwd/pw.x ?

Reply:
Apologies. You are absolutely correct.

Maybe it would be useful to have the list of binaries be strings and not Paths to prevent confusion?